src/writing-your-own-application/index.md

```diff
@@ -1 +1,4 @@
 # Writing Your Own Application
+
+To create a new project, [esp-generate](../getting-started/tooling/esp-generate.md) is used. 
```

Comment:
I really like the brevity here but as a user I probably would expect to see a little bit more?

But we already cover the basics in tooling/esp-generate - so not sure what to suggest here